data/locale/fr.ts

```diff
@@ -584,7 +584,7 @@ If you&apos;re interested in translating LMMS in another language or want to imp
     <name>BBTrack</name>
     <message>
         <source>Beat/Bassline %1</source>
-        <translation type="unfinished">Ryhtme ou ligne de basse %1</translation>
+        <translation type="unfinished">Rythme ou ligne de basse %1</translation>
```

Comment:
I've always assumed that too but I don't know... There are other instances of this too, such as this file:
https://github.com/LMMS/lmms/blob/master/data/locale/cs.ts